themes/godotengine/pages/showcase/city-game-studio.htm

```diff
@@ -0,0 +1,73 @@
+title = "Showcase | City Game Studio"
+url = "/showcase/city-game-studio"
+layout = "showcase-item"
+description = "City Game Studio showcase page"
+is_hidden = 0
+==
+{##}
+{% put title %} City Game Studio {% endput %}
+{% put author %} Binogure Studio {% endput %}
+{% put author_url %} https://hivetime.twolofbees.com/ {% endput %}
```

Comment:
This is probably a wrong URL, right?

Reply:
Good catch, fixed :slightly_smiling_face: